components/mpas-seaice/src/analysis_members/mpas_seaice_ridging_diagnostics.F

```diff
@@ -258,9 +258,7 @@ subroutine seaice_compute_ridging_diagnostics(domain, instance, timeLevel, err)!
            tracersPool
 
       real(kind=RKIND), dimension(:), pointer :: &
-           levelIceAreaAverage, &
            ridgedIceAreaAverage, &
-           levelIceVolumeAverage, &
```

Comment:
Is it not possible to keep diagnostics in the analysis members, so that they are available in case someone wants to look at them, but turn off the output (maybe in Registry) for the default case?  The level ice area and volume are the fundamental tracers in the model; ridged ice area and volume is a derived diagnostic.

Reply:
This is in the ridged ice analysis member.  levelIceAreaCell and levelIceVolumeCell can be switched on in output by adding them to, e.g., the "timeSeriesStatsMonthlyOutput" section in buildnml as is done for other model variables: https://github.com/E3SM-Project/E3SM/blob/5d5f15c93247cd318b3f89d964bdc85fc41d480f/components/mpas-seaice/cime_config/buildnml#L755-L866